tests/test_review_conference_talk.py

```diff
@@ -0,0 +1,99 @@
+
+<a name="p25"></a>Through the Prophet Joseph Smith, the Lord revealed the scripture which we have already heard from Elder Banks this evening—“that the thing which will be of the most worth unto you will be to declare repentance unto this people, that you may bring souls unto me.” ([D&C 15:6](https://www.churchofjesuschrist.org/study/scriptures/dc-testament/dc/15.6?lang=eng#p6); [D&C 16:6](https://www.churchofjesuschrist.org/study/scriptures/dc-testament/dc/16.6?lang=eng#p6).)
+
+<a name="p26"></a>Over the years we have been so impressed by the thousands of missionaries we have seen at the Missionary Training Center, at Ricks College, and elsewhere, who have demonstrated their willingness to serve their missions—and some of them, at great personal sacrifice.
+
+<a name="p27"></a>Brethren, may it be that in our lives generally, and in our priesthood responsibilities specifically, we, like David, my good friend, will set the kind of example so that our bishops will not have to wonder or worry about what we have been doing the Saturday night before.
+
+<a name="p28"></a>I am grateful for sons who still come with me to general priesthood sessions. You young brethren who are not with your fathers tonight, for whatever reason, can decide right now that when you are blessed with sons of your own, you will bring them to the general priesthood sessions wherever they may be broadcast.
+
+<a name="p29"></a>As I look up into the balcony tonight, I see some of you young men who are seated with your fathers, and I remember—I remember that first time so long ago. Dad passed away four years ago, and especially at general priesthood session time I am reminded more forcefully of how much we miss him. May we strive never to do anything that would embarrass or disappoint our Father in Heaven or our parents, and it will help make more of our memories to be good ones, because good memories constitute the “one Garden of Eden out of which we need never be cast.”
+
+<a name="p30"></a>Young brethren, we respect you. We have confidence that you will rise to the best that is in you, and we love you.
+
+<a name="p31"></a>Our Heavenly Father lives. He also loves you and even knows you by name. Jesus is the Christ, and this is His church, led by the living prophets who are presiding at this general priesthood session. I share this testimony in the holy name of Jesus Christ, amen.
+
+"""
+
+
+def check_text(markdown_content: str) -> Any:
+    # result check variable boolean
+    result_check = []
+    markdown_content = clean_text(markdown_content)
+
+    # Search for the position of "abstract" (case insensitive)
+    text_match = re.search(r'<a name="p(\d+)"><\/a>', markdown_content, re.IGNORECASE)
+
+    if text_match:
+        # Check content after the "text" text
+        result_check.append("True")
+
+    return result_check
+
+
+def test_review_conference() -> None:
+    """It returns a valid Document."""
+    result = check_text(mark_text)
```

Comment:
Tests need to call functions in files in the models directory. They should not generally call other functions in the test file itself. Your test file should include an HTML document, then call load_conference_talk imported from load_conf.py, passing the HTML document, and assert that the resulting page_content no longer contains the text that you don't want it to contain.